enhancements/operator-hub-check.md

```diff
@@ -0,0 +1,215 @@
+| operator bundle supports k8s `=> 1.16 <= 1.22` and uses `apiextensions/v1beta1` ÒR `admissionregistration.k8s.io/v1beta1` | warning |
+
+**Why we need to check if the api v1 is not used instead of only check if it has any CRD or Webhook with the deprecated `v1beta1`?**
+		
+The operator author might provide a solution that works for both scenarios which means that will work when installed in the previous version of `Kubernetes < 1.16` and also in the upper versions `=> 1.22`. However, for the operator works well in any Kubernetes version `>=1.22`, we know that it is mandatory it has the latest APIs versions `v1`. See [here](https://github.com/kubernetes-sigs/kubebuilder/blob/v3.0.0-beta.0/test/e2e/v3/plugin_cluster_test.go#L99-L101), for example, that in the e2e test for Kubebuilder, we check what is the cluster version to apply and test the API versions accordingly. And then, note that we can find both APIs in the webhooks fields of the CSV as well such as:
+
+```
+  webhookdefinitions:
+  - admissionReviewVersions:
+    - v1beta1
+    - v1   
+``` 
+
+**IMPORTANT** 
+
+> Kubernetes version `1.22` was not released until now which means that the deprecated APIs still supported for all clusters. Also, at the same way none OpenShift version which is using Kubernetes API `1.22` was released. Then, we cannot raise errors until it happens for the scenarios where we are `minKubeVersion` is empty and we are assuming all possible versions.
+> However, for the checks where the `--optional-values="k8s=1.22"` was used we know that the goal is to verify if the operator bundle will work òn `1.22` then, we can already raise the `error` result. 
+
+Then, it means that:
+- `--optional-values="kubeVersion=value"` flag with a value `=> 1.16 <= 1.22` the validator will return result as warning.
+- `--optional-values="kubeVersion=value"` flag with a value `=> 1.22` the validator will return result as error.
+- `minKubeVersion >= 1.22` return the error result.
+
+**Currently (Before `1.22` release)**
+- `minKubeVersion` empty/nil return the warning result.
+
+**After `1.22` k8s release**
+- `minKubeVersion >= 1.22 or empty/nil`  return the error result.
+
+### Risks and Mitigations
+
+The Kubernetes `1.22` release will impact the Pipeline. After the implementation of the check: 
+
+**After `1.22` k8s release**
+- `minKubeVersion >= 1.22 or empty/nil`  return the error result.
+
+All bundles which are using the no longer supported Kubernetes API will start to fail in the above check which indeed is the purpose of this proposal. However, the risk still solved since the Pipeline can indeed inform a Kubernetes version lower such as `--optional-values="k8s=1.21"` if required until all projects are properly update.
+
```

Comment:
Hi @bparees, 

Also, see that the Pipeline scenario/risk and mitigations are also covered properly now.